neurokit2/ecg/ecg_respiration.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import scipy.interpolate
+
+from ..signal.signal_formatpeaks import _signal_formatpeaks_sanitize
+from ..signal import signal_detrend
+
+
+def ecg_respiration(peaks, sampling_rate=1000, show=False):
```

Comment:
that's nice, but have you seen `ecg_rsp`? 😅

Reply:
oooooops my sick brain missed it 😆